modules/diff.js

```diff
@@ -115,6 +117,15 @@ const getGitDiff = async ({ product, base, head = 'master', dev, prod }) => {
   if (contributors.size > 1) {
     r.attachments[0].blocks[0].text.text += `\n\nContributors: ${Array.from(contributors).join(', ')}`
   }
+
+  // link to demos calendar
+  if (demos) {
+    const {day, link, events } = demos
+    r.attachments[0].blocks[0].text.text += `\n\n*Demos* on <${link}|${day}>`
+    events.forEach(({ timeSloth }) => {
+      r.attachments[0].blocks[0].text.text += `\n\t• ${timeSloth}`
```

Comment:
what is `•`?

Reply:
should work for mrkdwn, I know, but no  ... I remeber the blocks being more elaborated for list and list items... 🤷 
![image](https://user-images.githubusercontent.com/53827690/102667433-60fbb180-4157-11eb-80eb-.586d425025eb.png)

Reply:
:trollface: